### DISCUSSION TITLE
Changes made to Overview page<title>

### DISCUSSION THREAD

**Discussion by jccviking** [community/community#114295] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

For whatever reason, GitHub changed the behavior of the Overview page.  It used to be, when I clicked on one of the "date boxes", the Contribution Activity would be limited to the clicked date. Now, the Contribution Activity is limited to the month containing the date I clicked.  In the attached capture, I clicked on March 23rd but the Contribution Activity is showing all contributions for the month of March instead of the contributions for March 23rd.

![2024-03-24_12-35-56](https://github.com/community/community/assets/17817855/edb19222-7b4e-48ae-90fc-dee03ac8dd68)

I'm puzzled as to why they would change such a thing.  I doubt most users would consider this a good change.

JCC

**Comment by Kakar21** [comment]
I can't even click on it anymore 😂

**Comment by jccviking** [comment]
You're right!  I thought it was aligning with the "clicked month" but clicking has no effect.

Again, begs the question: Why remove perfectly good functionality?

**Reply by Kakar21** [reply]
I really don't know. Maybe they are implementing a new feature for it. But I don't think they will answer this question here :/

**Comment by Kakar21** [comment]
Hey, the feature is working again! 🙌

**Reply by Kakar21** [reply]
I mean this one

**Comment by jccviking** [comment]
>> Hey, the feature is working again!
 
Cool!  Thank you so much
 
Jccviking.

**Reply by Kakar21** [reply, marked as answer]
no problem, could you mark my answer as solution?

**Reply by jccviking** [reply]
Done!  Thanks again!
